_posts/2020-07-21-cube-functionality-to-kartothek.markdown

```diff
@@ -168,7 +168,7 @@ dataset:
 ```
 
 Here we use **geodata** cube to get some information.
-```python
+```shell
```

Comment:
I only meant to change THIS particular occurrence. The other examples are Python code.

Reply:
done.